autoprotocol/protocol.py

```diff
@@ -1185,6 +1207,8 @@ def liquid_handle_dispense(
         default_num_dispense_chips_in_source: int = 1
         default_max_num_dispense_chips: int = 12
         remaining_num_chips_to_specify = default_max_num_dispense_chips
+        if device == "x_mantis":
+            rows = 1
```

Comment:
should we assign this for the protocol writer or should we validate rowx,columns vs device and like error at them?

Reply:
I usually lean towards auto-assigning based on the capabilities of the device, but if we want to give more flexibility, I'm fine with a validation & erroring at them!

Reply:
I ended up just giving the protocol writer that flexibility and adding in a validation in `util.py`. Should be in the commit just now!